src/format.h

```diff
@@ -4,12 +4,12 @@ Copyright (C) 2016-2023 Deep Genomics Inc. All Rights Reserved.
 #ifndef __GENOME_KIT_FORMAT_H__
 #define __GENOME_KIT_FORMAT_H__
 
-#include <fmt/core.h>
+#include <format>
 
 BEGIN_NAMESPACE_GK
 
 // Returns string formatted according to sprintf(fmt, ...) conventions.
-using fmt::format;
+using std::format;
```

Comment:
I think somewhere this went off the rails.  This file is supposed to select the correct format implementation so you don't have all the std/fmt:: prefixes everywhere else.  if we're going to explicitly reference the namespace, we can delete this